week-2/Homework/mandatory/1-fix-functions.js

```diff
@@ -33,12 +33,12 @@ function first5() {
   let numbers = [1, 2, 3, 4, 5, 6, 7, 8];
   let sliced;
 
-  return sliced;
+  return (sliced = numbers.splice(0, 5));
 }
 
 function get3rdIndex(arr) {
   let index = 3;
-  let element;
+  let element = arr[3];
```

Comment:
Hi Bashir here you could use the index variable as it has already been assigned a value of 3
```suggestion
  let element = arr[index];
```